content/plugins/commons-chunk-plugin.md

```diff
@@ -182,6 +187,23 @@ new webpack.optimize.CommonsChunkPlugin({
 })
 ```
 
+To add further control to an implicit common vendor chunk, you can additional logic before the `node_modules` context check to prevent
+specific modules from being included in the vendor chunk (such as stylesheets you want to keep in the original chunk):
```

Comment:
The rationale for adding this example comes from a gotcha from using the implicit vendor chunk with the `ExtractTextPlugin`.

`ExtractTextPlugin` separates your CSS into its own separate CSS bundle - but when combined with implicit common vendor chunk logic alone, it will generate another separate vendor-only CSS bundle, so you end up with `vendor.css` and an `application.css` for example.

This separated vendor CSS chunk might not be immedately obvious to users (which is confusing when you intend to just have the single separate CSS chunk(`application.css`) from `ExtractTextPlugin`). It also complicates the ordering of CSS `require` statements, since the CSS modules will be split up across the vendor and original chunk - this can affect style precedence. I think most `ExtractTextPlugin` users would prefer to have 1 `application.css` chunk instead of `application.css` + `vendor.css` chunks.

It took me some time to debug why our styles were being affected in production after we changed from an explicit to implicit vendor chunk, so I thought this might be worth pointing out in case anybody else gets caught up by their `ExtractTextPlugin` setup. (Having the function arguments for `minChunks` also helps a lot here to too!)

I'm fine with dropping this example but I do think it would help users who might have a similar setup with `ExtractTextPlugin`. Let me know what you think @simon04!